tests/test_lab_data_list.py

```diff
@@ -0,0 +1,129 @@
+# Standard
+import pathlib
+import re
+import textwrap
+import time
+
+# Third Party
+from click.testing import CliRunner
+
+# First Party
+from instructlab import lab
+from instructlab.cli.data.list import extract_model_and_timestamp
+
+
+def normalize_table_output(table_output):
```

Comment:
What's the reason for this normalization?

Reply:
actually, based on the textwrap.dedent to print the expected_output(table), seems it will cause more format issue between `result.output` and `expected_output`(e.g. extra spaces would be different for different OS), normalize_table_output is to help format it.  That's why the ubuntu failed several times.
Can it simply based on the entry to check(not focus on the table format)? That might be simple.

Reply:
> actually, based on the textwrap.dedent to print the expected_output(table), seems it will cause more format issue between `result.output` and `expected_output`(e.g. extra spaces would be different for different OS),

If dedent is causing an issue, you can leave it out and keep the formatting as needed in the test.




> Can it simply based on the entry to check(not focus on the table format)? That might be simple.

I am not quite sure what this means.  Some sort of fuzzy checking of the data being there without getting every space of the formatting would be good enough.  But it is nice when you can see the format in the test.  That way when someone changes the code, you'll be able to see the changes clearly in the updated test.

Reply:
Thank you for the advice. Just updated. Keep the format in expected output, but simply compare content from result and expect. feel free to take a look.